tensorly/__init__.py

```diff
@@ -87,25 +88,19 @@
     lstsq,
     qr,
     kr,
-    partial_svd,
     stack,
     eps,
     finfo,
     matmul,
     index_update,
     check_random_state,
-    randomized_svd,
     randn,
-    randomized_range_finder,
     log,
     log2,
     exp,
     sin,
     cos,
     tan,
-    asin,
-    acos,
-    atan,
```

Comment:
Why did you remove those (as well as the hyperbolic versions down)?

Reply:
I believe functions are needed in some downstream libraries, at least by @j6k4m8, in which case I'd rather leave them to not break other's code -- @j6k4m8 do you need `asinh` as well as `arcsinh`?

Reply:
I don't have an attachment to these names in particular; I added `aXXX` along with `arcXXX` versions to maximize compatibility (since the only cost of supporting them is a string match). [Here's an example](https://github.com/tensorly/tensorly/pull/398/files#diff-f605db052e892b4c76b249d0e5751b9b747bee9d1e5d825adab44e43d833ca18R253) of the lookup for the function calls from that PR.

I don't mind if you want to remove them, although a more appropriate treatment might be to simply point `arccos` (for example) to `acos` in the [base backend implementation](https://github.com/tensorly/tensorly/pull/398/files#diff-3e1da4d92e2d739adfbcf25b91c53643851f4da8d7948b314f4e74ff8d93a748R1548); i.e.:

```python

    def arccos(self, x):
       return self.acos(x)

```

...which obviates the need altogether. But I also don't mind just removing them if you prefer to be numpy-API-exclusive!

Reply:
Thanks @j6k4m8. I'll add wrappers as you suggest for now. We'll have plenty of big changes in the next version already!